lib/config/configuration.go

```diff
@@ -635,6 +635,7 @@ func applyAuthConfig(fc *FileConfig, cfg *service.Config) error {
 	}
 
 	if fc.Auth.MessageOfTheDay != "" {
+		cfg.Auth.Preference.SetOrigin(types.OriginConfigFile)
```

Comment:
I believe this was missed in de9bdf086dd5471278ddc7ab1fb6d9a66c9f9882.

Reply:
What if the origin is already `OriginDynamic`? `MessageOfTheDay` seems less consequential than the rest of the auth preferences, I'm not sure it makes sense to overwrite the origin for this, but I don't know much about what these origin tags are used for